modules/basic-skills/src/views/full/slot.jsx

```diff
@@ -98,14 +102,34 @@ export class Slot extends React.Component {
   }
 
   isFormValid() {
+    const isTurnExpiryAndRetryAttemptsValid = this.isTurnExpiryAndRetryAttemptsValid(
+      this.state.turnExpiry,
+      this.state.maxRetryAttempts
+    )
+
     return (
+      isTurnExpiryAndRetryAttemptsValid &&
       this.state.selectedIntentOption &&
       this.state.selectedSlotOption &&
       this.state.contentElement &&
       this.state.notFoundElement
     )
   }
 
+  isTurnExpiryAndRetryAttemptsValid(turnExpiry, retryAttempts) {
+    return turnExpiry !== 0 || retryAttempts !== 0
```

Comment:
This only checks if one of turn expiry or retry attempts is equal to 0, not both, but the error messages says that it is invalid to have both set at 0

Reply:
@michaelmass I'm not sure about this one, what's the validation you are trying to do here?